python/apps/taiga/src/taiga/workflows/api/__init__.py

```diff
@@ -120,3 +121,49 @@ async def create_workflow_status(
         color=form.color,
         workflow=workflow,
     )
+
+
+################################################
+# update workflow status
+################################################
+
+
+@routes.stories.patch(
+    "/projects/{project_id}/workflows/{workflow_slug}/statuses/{status_slug}",
+    name="project.workflowstatus.update",
+    summary="Update workflow status",
+    response_model=WorkflowStatusSerializer,
+    responses=ERROR_404 | ERROR_422 | ERROR_403,
+)
+async def update_workflow_status(
+    request: AuthRequest,
+    form: UpdateWorkflowStatusValidator,
+    project_id: B64UUID = Query(None, description="the project id (B64UUID)"),
+    workflow_slug: str = Query(None, description="the workflow slug (str)"),
+    status_slug: str = Query(None, description="the status slug (str)"),
+) -> WorkflowStatus:
+    """
+    Updates a workflow status in the given project workflow
+    """
+    workflow = await get_workflow_or_404(project_id=project_id, workflow_slug=workflow_slug)
+    await check_permissions(permissions=UPDATE_WORKFLOW_STATUS, user=request.user, obj=workflow)
+    status = await get_status_or_404(workflow_id=workflow.id, status_slug=status_slug)
+
+    return await workflows_services.update_workflow_status(
+        status=status,
+        # TODO: use `values=await form.cleaned_dict(request)` instead
```

Comment:
why this TODO?

Reply:
I was thinking in adapting this method, or a new one, to centralize similar patch params. But yes, it's not prioritary right now and there are other places in a similar scenario without the tag. We can take this decisition in the future, if it comes...
I'll remove the TODO.